base/genimport/importer.go

```diff
@@ -269,14 +274,93 @@ func removeAllFilesInDirExcept(o *Output, dir string, except_list []string) {
 }
 
 func createPluginGoModFile(o *Output, pkgpath string, dir string) string {
+	newLines := [][]byte{
+		[]byte("module gomacro.imports/" + pkgpath + "\n"),
+	}
+
+	// TODO: Figure out a better way to guess the dir
```

Comment:
Is this TODO still necessary?  If so it would be helpful to clarify what it means.  Is this just for if we add support for `import "../mypkg"`?

Reply:
Yeah, this is just add support for local paths (or replace directives). I'll clarify the comment.